src/Utils.ps1

```diff
@@ -176,6 +176,14 @@ function Add-PoshGitToProfile {
         $profileContent = "`nImport-Module '$ModuleBasePath\posh-git.psd1'"
     }
 
+    # Make sure the profile path exists
+    $profileDir = Split-Path -Parent $profilePath
```

Comment:
No problem, I moved the -Parent parameter to the end.

I also looked at using -LiteralPath with Split-Path, but it does not have an explicit -LiteralPath parameter, however based on the below test it looks like it already implicitly treats the path literally.

```
C:\test[1]> Get-Item -LiteralPath c:\test[1]\File[1].txt
    Directory: C:\test[1]
Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----         3/2/2017   3:18 AM              0 File[1].txt

C:\test[1]> Get-Item -Path c:\test[1]\File[1].txt

C:\test[1]> Split-Path -Path c:\test[1]\File[1].txt -Parent
c:\test[1]
```

I created a directory and file with names that both need to be interpreted literally.

- Get-Item with -LiteralPath works fine

- Get-Item with -Path fails to retrieve the file

- Split-Path with -Path however still correctly parses the path to retrieve the parent

So, I think Split-Path is good with just -Path.  Looking through the other instances of Split-Path in the code, I figured I would keep it consistent and not explicitly state -Path, but let me know if you want me to change it.  It's no problem.